src/styles/cards/catalog.scss

```
@@ -0,0 +1,35 @@
+.catalog {
+  font-family: $body_font;
+  font-size: 10px;
+  padding: 50px 40px;
+
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  gap: 48px;
+  row-gap: 46px;
```

We can write it one property
```suggestion
  gap: 46px 48px;
```